{
  "version": 2,
  "status": "error",
  "message": "stack trace: (most recent call last)\nidentity.nim(3, 5) identity\n\u001b[1midentity.nim(3, 5) \u001b[0m\u001b[31mError: \u001b[0munhandled exception: myValueError [ValueError]\u001b[36m\u001b[0m\u001b[0m\n",
  "tests": []
}
